Create README

sharknoon opened the pull request:
- added initial readme

sharknoon committed:
Create README.md

Added initial readme

artheus commented:
The overall feel here is a "docker tutorial".
I would much rather provide a link to https://docs.docker.com/engine/reference/run/
and rather provide more details about creating word-data, server.properties files and directories for mounting im the images. To provide persistent configuration and world data.

sharknoon committed:
README changes

- updated Modpack table
- removed some docker instructions

sharknoon committed:
Further refining of the README

- swapped the order of the versions from All the Mods 6
- borrowed some sections for contribution and credit from a source, where I think the author is okay with it ;) Just kidding, it is from the same author, but from another repository
- added volume section

sharknoon commented:
I changed some things, what do you think? What can be improved?
